pyaedt/emit_core/EmitConstants.py

```diff
@@ -97,8 +97,10 @@ def unit_type():
 }
 """Valid units for each unit type."""
 
-desktop_path = os.environ.get("ANSYSEM_ROOT232")
-if desktop_path and sys.version_info < (3, 8):
+desktop_path = os.environ.get("ANSYSEM_ROOT241")
```

Comment:
@apollo-crum Can we use _aedt_version or something else within the Emit() class to retrieve this path so it keeps up with version changes automatically?

Reply:
I'm thinking I'll just remove it actually.  Unless someone knows why this check is needed?  @myoung301 @jsalant22

Reply:
@apollo-crum I think it was needed to avoid calling unit_type() if only 23.1 or older is installed. So maybe change it to match the code used in emit.py:

if self._aedt_version >= "2023.1":